lua/nvim-treesitter/install.lua

```diff
@@ -62,13 +98,24 @@ function M.install_parser(lang)
     return
   end
 
+  local err, dl_command = create_download_command(repository.url)
+  if err ~= nil then
+    api.nvim_err_writeln(err)
+    return
+  end
+
   print("Downloading parser...")
-  fn.system(create_download_command(repository.url))
+  fn.system(dl_command)
```

Comment:
If dowmloading fails, this call might return an error and further steps might cause unexpected things.